mne/realtime/tests/test_mockclient.py

```diff
@@ -174,9 +175,12 @@ def test_find_events():
     rt_client = MockRtClient(raw)
     rt_epochs = RtEpochs(rt_client, event_id, tmin, tmax, picks=picks,
                          stim_channel='STI 014', isi_max=0.5,
-                         find_events=find_events)
+                         find_events=find_events, verbose='WARNING')
```

Comment:
Let's revert these. Typically we only set `verbose` in testing to improve / test outputs. Locally you can run with `MNE_LOGGING_LEVEL=warning pytest ...` if you do not want to permanently switch to `warning` level

Reply:
ok, they're gone.